### PR TITLE
Fixesforchandler

### PR DESCRIPTION
**Pull request by claremboyle**

### CODE REVIEW

**Comment by claremboyle** [comment]
this pull request of in place of #10 and includes the changes you asked for

**Comment by claremboyle** [comment]
@Figler more changes are needed still. I will comment when those are ready.

**Comment by chan4est** [comment]
@claremboyle so don't approve just yet?

File: src/test/java/edu/ucsb/cs56/pconrad/SparkDemo01Test.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		System.setOut(originalOut);
+		System.setErr(originalErr);
+	}
+
+	@Test
+	public void test_out() {
+		assertTrue(true);
+	}
+	
+}
```
Comment (chan4est) [inline src/test/java/edu/ucsb/cs56/pconrad/SparkDemo01Test.java]: You still need to properly unit test your code before pushing into master. Send the PR once you've gone ahead and fully tested and double checked all your code.

**Review by chan4est — CHANGES_REQUESTED** [review]

File: src/test/java/edu/ucsb/cs56/pconrad/SparkDemo01Test.java (hunk 1 above)
Comment (claremboyle) [reply src/test/java/edu/ucsb/cs56/pconrad/SparkDemo01Test.java]: Yeah, ik. Conrad was showing me how to branch 😅 and then send the branch to our main App. But we just changed one thing, and realize that after I sent the request. Sorry.

**Comment by claremboyle** [comment]
for issue #2 and #5 as well as modifying the documents with requested changes

**Pull request closed without merging**